doc/api/deprecations.md

```diff
@@ -1050,6 +1050,16 @@ deprecated along with the undocumented aliases `crypto.prng()` and
 `crypto.rng()` in favor of [`crypto.randomBytes()`][] and will be removed in a
 future release.
 
+<a id="DEP00XX"></a>
+### DEP00XX: Legacy URL API
+
+Type: Documentation
+
+The [Legacy URL API][] is deprecated in favor of the newer standardized
+[WHATWG URL][] implementation. This includes uses of `url.format()`,
```

Comment:
`url.format()` should be qualified to not include the overload that takes `URL` objects.

Reply:
I'm not sure it should. I added that strictly as a convenience. The intent here is to move people away from the non-standard API and keeping that one variant un-deprecated would not make sense. Folks really should be using the `href` property on the `URL` object instead.